include/wasm.hh

```diff
@@ -11,6 +11,10 @@
 #include <limits>
 #include <string>
 
+#ifndef WASM_C_API
+#define WASM_C_API
```

Comment:
Is nothing needed for C++? And if not, why do we bother defining the macro?

Reply:
I'm pretty sure C++ needs the same definition.

Reply:
I just forgot to update the one in `wasm.hh` when I made this change: https://github.com/WebAssembly/wasm-c-api/pull/77/commits/b02664964794e56d2965edf44551ac7eaf5d4fb2.

The idea is that if you need it, you define `WASM_C_API=whatever` on the command line rather than expecting the header file to know which side of the import/export it's on.